racket/src/racket/src/schvers.h

```diff
@@ -16,7 +16,7 @@
 #define MZSCHEME_VERSION_X 7
 #define MZSCHEME_VERSION_Y 4
 #define MZSCHEME_VERSION_Z 0
-#define MZSCHEME_VERSION_W 1
+#define MZSCHEME_VERSION_W 3
```

Comment:
Also need to update "info.rkt" in the "base" package.

Reply:
Fixed in 2faf3b6cfc5d7d14fa2b3b26b39289c7e6b1a59a